test/expected/chunk_utils_compression.out

```diff
@@ -0,0 +1,138 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
We recommend running tests as a low-privileged user and only switch to super-user when necessary.